testsuite/Gemfile

```diff
@@ -29,3 +29,4 @@ gem 'redis', '~> 5.0'
 # or build and install from https://github.com/openSUSE/twopence
 gem 'twopence', "~> 0.4"
 gem 'pg', "~> 1.4"
+gem 'public_suffix', '~> 4.0'
```

Comment:
Why we need this new gem dependency?

Reply:
It is a dependency already, but we need it to be ~> 4.0 or it pulls a newer version which depends on a newer ruby version which opensuse 15.4 does not have.

Reply:
See https://github.com/weppos/publicsuffix-ruby/blob/main/CHANGELOG.md#500